src/framework/mpas_dmpar.F

```diff
@@ -17,6 +17,8 @@
 !-----------------------------------------------------------------------
 module mpas_dmpar
 
+!#define MPAS_DEBUG
```

Comment:
I think the commented-out definition of `MPAS_DEBUG` should be removed. `MPAS_DEBUG` is a macro that is defined in the top-level `Makefile` when compiling MPAS with `DEBUG=TRUE`, and so we should not override that choice here.